GPU/GPUbenchmark/Shared/Utils.h

```diff
@@ -157,14 +169,19 @@ inline float computeThroughput(Test test, float result, float chunkSizeGB, int n
 {
   // https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html
   // Eff_bandwidth (GB/s) = (B_r + B_w) / (~1e9 * Time (s))
-
+ 
   return 1e3 * chunkSizeGB * ntests / result;
 }
 
 template <class chunk_t>
-inline size_t getBufferCapacity(float chunkReservedGB)
+inline size_t getBufferCapacity(float chunkSizeGB, int prime)
 {
-  return (static_cast<size_t>(GB * chunkReservedGB) & 0xFFFFFFFFFFFFF000) / sizeof(chunk_t);
+  auto chunkSizeBytes = static_cast<size_t>(GB * chunkSizeGB) & 0xFFFFFFFFFFFFF000;
```

Comment:
I think you should divide by `sizeof(chunk_t)` first, and then reduce it if is 0 modulo prime.

Reply:
ofc!